exchanges/bybit/bybit_websocket.go

```diff
@@ -0,0 +1,481 @@
+			if err != nil {
+				by.Websocket.DataHandler <- order.ClassificationError{
+					Exchange: by.Name,
+					OrderID:  orderID,
+					Err:      err,
+				}
+			}
+			var oSide order.Side
+			oSide, err = order.StringToOrderSide(data.Side)
+			if err != nil {
+				by.Websocket.DataHandler <- order.ClassificationError{
+					Exchange: by.Name,
+					OrderID:  orderID,
+					Err:      err,
+				}
+			}
+			var oStatus order.Status
+			oStatus, err = stringToOrderStatus(data.OrderStatus)
+			if err != nil {
+				by.Websocket.DataHandler <- order.ClassificationError{
+					Exchange: by.Name,
+					OrderID:  orderID,
+					Err:      err,
+				}
+			}
+
+			p, err := by.extractCurrencyPair(data.Symbol, asset.Spot)
+			if err != nil {
+				return err
+			}
+
+			by.Websocket.DataHandler <- &order.Detail{
+				Price:           data.Price,
+				Amount:          data.Quantity,
+				ExecutedAmount:  data.CumulativeFilledQuantity,
+				RemainingAmount: data.Quantity - data.CumulativeFilledQuantity,
+				Exchange:        by.Name,
+				ID:              orderID,
+				Type:            oType,
+				Side:            oSide,
+				Status:          oStatus,
+				AssetType:       asset.Spot,
+				Date:            data.OrderCreationTime.Time(),
+				Pair:            p,
+				ClientOrderID:   data.ClientOrderID,
+			}
+			return nil
+		case wsOrderFilledStr:
+			// already handled in wsOrderStr case
+			return nil
+		}
+	}
+
+	t, ok := multiStreamData["topic"].(string)
+	if !ok {
+		log.Errorf(log.ExchangeSys, "%s Received unhandle message on websocket: %v\n", by.Name, multiStreamData)
+		return nil
+	}
+
+	switch t {
+	case wsOrderbook:
+		var data WsOrderbook
+		err := json.Unmarshal(respRaw, &data)
+		if err != nil {
+			return err
+		}
+		p, err := by.extractCurrencyPair(data.OBData.Symbol, asset.Spot)
+		if err != nil {
+			return err
+		}
+
+		err = by.wsUpdateOrderbook(&data.OBData, p, asset.Spot)
+		if err != nil {
+			return err
+		}
+
+	case wsTrades:
+		if !by.IsSaveTradeDataEnabled() {
+			return nil
+		}
+		var data WsTrade
+		err := json.Unmarshal(respRaw, &data)
+		if err != nil {
+			return err
+		}
+
+		p, err := by.extractCurrencyPair(data.Parameters.Symbol, asset.Spot)
+		if err != nil {
+			return err
+		}
+
+		side := order.Sell
+		if data.TradeData.Side {
+			side = order.Buy
+		}
+
+		return trade.AddTradesToBuffer(by.Name, trade.Data{
+			Timestamp:    data.TradeData.Time.Time(),
+			CurrencyPair: p,
+			AssetType:    asset.Spot,
+			Exchange:     by.Name,
+			Price:        data.TradeData.Price,
+			Amount:       data.TradeData.Size,
+			Side:         side,
+			TID:          data.TradeData.ID,
+		})
+
+	case wsTicker:
+		var data WsSpotTicker
+		err := json.Unmarshal(respRaw, &data)
+		if err != nil {
+			return err
+		}
+
+		p, err := by.extractCurrencyPair(data.Ticker.Symbol, asset.Spot)
+		if err != nil {
+			return err
+		}
+
+		by.Websocket.DataHandler <- &ticker.Price{
+			ExchangeName: by.Name,
+			Bid:          data.Ticker.Bid,
```

Comment:
Please add in high low close open volume and any other supported missing fields.

Reply:
This information is missing. Please check this.
https://bybit-exchange.github.io/docs/spot/#t-websocketv2bookticker

@shazbert